src/main/java/com/google/api/codegen/config/GapicInterfaceConfig.java

```diff
@@ -286,31 +298,6 @@ private static SmokeTestConfig createSmokeTestConfig(
     }
   }
 
-  private static <T> List<T> createMethodConfigs(
```

Comment:
This method has been moved to a static method in GapicProductConfig, as it is used by both GapicInterfaceConfig and DiscoGapicInterfaceConfig

Reply:
Update: removing this method, but keeping its method overload below.